### PR TITLE
[testcase] add user test cases

### PR DESCRIPTION
**Pull request by jude-zhu**

### CODE REVIEW

File: tests/tck/features/user/User.feature [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      ALTER USER user2 WITH PASSWORD "pwd1"
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      ALTER USER user2 WITH PASSWORD "pwd2"
```
Comment (bright-starry-sky) [inline tests/tck/features/user/User.feature:68]: It's better to add a validation at here. to check the password was changed successfully. WDYT?
for example 
```
CHANGE PASSWORD user2 FROM "pw2" TO "pwd2"   ---> expected fail
CHANGE PASSWORD user2 FROM "pwd2" TO "pwd3" ---> expected successful
```

**Comment by bright-starry-sky** [comment]
Generally good for me. nice job.

File: tests/tck/features/user/User.feature [hunk 2]
```diff
@@ -0,0 +1,270 @@
+Feature: User & privilege Test
+
+  Background: Prepare space
```
Comment (yixinglu) [inline tests/tck/features/user/User.feature:3]:
```
Background:
   Given an empty graph
```

**Review by bright-starry-sky — APPROVED** [review]
Well done.

**Review by yixinglu — APPROVED** [review]

**Pull request merged**